src/redux/books/books.js

```diff
@@ -0,0 +1,16 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const booksReducers = (state = [], action) => {
+  switch (action.type) {
+    default:
+      return state;
+  }
+};
+
+export const addBook = () => ({ type: ADD_BOOK });
+
+export const removeBook = () => ({ type: REMOVE_BOOK });
+
```

Comment:
- [ ] From the project you can see we are trying to add and remove books. That means apart from the action types, you still need to include the payloads that would be used to work on the store.

  ```
  export const addBook = (payload) => ({
    type: ADD_BOOK,
    payload,
  });
  
  export const removeBook = (payload) => ({
    type: REMOVE_BOOK,
    id: payload,
  });
  ```

Reply:
Resolved